lib/IRGen/IRGenModule.cpp

```diff
@@ -440,6 +440,7 @@ llvm::Constant *swift::getRuntimeFn(llvm::Module &Module,
     fn->setCallingConv(cc);
 
     if (llvm::Triple(Module.getTargetTriple()).isOSBinFormatCOFF() &&
```

Comment:
I think that this should use `useDllStorage()` instead of adding the negation check so that this is more similar to the previous case.